ui/widgets/my-apps/src/my-apps-widget.tsx

```diff
@@ -169,9 +170,18 @@ const Wrapped = () => {
   const { getTranslationPlugin } = useRootComponentProps();
 
   return (
-    <I18nextProvider i18n={getTranslationPlugin().i18n}>
-      <ICWidget />
-    </I18nextProvider>
+    <Router>
```

Comment:
you can remove this router from here

Reply:
the router is needed, else the 'my-apps' persists on every other page